HW12/hw_12_task_1.py

```diff
@@ -0,0 +1,31 @@
+# 1. Написати власний декоратор, задачею якого має бути друк назви функції і часу, коли вона була викликана. Декоратор має працювати для різних функцій однаково.
+
+import time
+
+def my_decorator(func):
+    # decorator logic
```

Comment:
nit: better to comment functions using `"""` format
```suggestion
    """Decorator logic"""
```